core/server/api/v0.1/posts.js

```diff
@@ -16,6 +16,55 @@ const Promise = require('bluebird'),
     ],
     unsafeAttrs = ['author_id', 'status', 'authors'];
 
+const mongo = require('../v2/utils/serializers/input/utils/mongo.js');
```

Comment:
Just for naming to be consistent, good to name it as `mongoUtils` as well like the other usage 😄